spec/controllers/list_items_controller_spec.rb

```diff
@@ -196,12 +197,18 @@
                   "contents" => [
                     list_item2.base_path,
                   ],
+                  "content_ids" => %w[
+                    456
```

Comment:
Should this be list_item2.content_id as per previous usage? similar for below

Reply:
Ah sure, do they not get accessed when calling the content_id method? I guess it should be calling the [content_id on tagged documents](https://github.com/alphagov/collections-publisher/blob/7818ba97bc2958ab0614ae4a863229cecd875752/app/models/list_item.rb#L16) ?